specification/monitor/resource-manager/microsoft.insights/preview/2018-06-01-preview/examples/createOrUpdateGuestDiagnosticSettingsAssociation.json

```diff
@@ -0,0 +1,42 @@
+{
+    "parameters": {
+        "resourceUri":"subscriptions/4abcdefgh-ijkl-mnop-qrstuvwxyz/resourceGroups/rhctestenv/providers/Microsoft.ClassicCompute/virtualMachines/rhctestenvV1PI",
```

Comment:
Shouldn't our default example be an ARM VM?

Reply:
Done